notes/login/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
 # Create your models here.
+
+from django.db import models
+
+
+class User(models.Model):
+    email = models.CharField(max_length=30)
```

Comment:
let's add validation for Email
docs: https://docs.djangoproject.com/en/4.2/ref/validators/#emailvalidator

```suggestion
from django.core.validators import EmailValidator

class User(models.Model):
    email = models.CharField(max_length=30, validators=[EmailValidator()])
```